wfe/wfe_test.go

```diff
@@ -1350,6 +1350,44 @@ func TestEmptyRegistration(t *testing.T) {
 	responseWriter.Body.Reset()
 }
 
+func TestNewRegistrationForbiddenWithAccountCreationDisabled(t *testing.T) {
+	wfe, _ := setupWFE(t)
+	_ = features.Set(map[string]bool{"AccountCreation": false})
+
+	key := loadPrivateKey(t, []byte(test2KeyPrivatePEM))
+	signer := newJoseSigner(t, key, wfe.nonceService)
+	// Reset the body and status code
+	responseWriter := httptest.NewRecorder()
+	// POST, Valid JSON, Key already in use
```

Comment:
This comment says "Key already in use", but it tests for the problem returned when a new registration would have been created. It looks like `TestNewRegistration409sWithAccountCreationDisabled` uses a key that is already in use. Is this comment wrong/misplaced or did I misunderstand something?

Reply:
Good catch. This was largely copy-pasted and I forgot to update the comments. The difference between the two tests is "test1Key" vs "test2key". Fixed up the comments.